documentation/guides/owner-journey-v1.md

```diff
@@ -191,7 +198,7 @@ kind: ConfigMap
 metadata:
   name: tmp-ca
   annotations:
-    service.beta.openshift.io/inject-cabundle: "true"
+    service.beta.OpenShift.io/inject-cabundle: "true"
```

Comment:
Should this be all lowercase?  Did you confirm that the annotation still works after changing the case?

Reply:
Good catch, I had find and replaced all instances of `openshift` (case insensitive search) and didn't realize it might appear in code.